website/views/pages/fleet-mdm.ejs

```diff
@@ -48,8 +48,9 @@
         <h3>Convention over configuration</h3>
         <p>Configure your users' computers with sensible defaults, or customize every traditional MDM feature exactly how you want it with Fleet's API, CLI, and webhooks.</p>
         <div purpose="checklist" class="flex-column d-flex">
-          <p>Set up and verify ongoing compliance with CIS standards (one click, 400+ queries, supported by Fleet)</p>
-          <p>Tasked with zero trust? Use sensible, enterprise-ready device health checks from the community</p>
+          <p>Migrate devices quickly with out-of-the-box features, or control every detail using a modern REST API</p>
```

Comment:
Good to bring up migrations, but what are the "features"?

Reply:
For MDM migration, Fleet has a guided wizard to walk end users through what they need to click to migrate from their old MDM to Fleet, implemented by Fleet Desktop.